dev/io.openliberty.jcache.internal/resources/io.openliberty.jcache.internal/resources/JCacheMessages.nlsprops

```diff
@@ -72,10 +72,10 @@ CWLJC0011_GET_CACHE_ERR=CWLJC0011W: Error encountered while retrieving the {0} J
 CWLJC0011_GET_CACHE_ERR.explanation=The cache was not retrievable due to the specified error.
 CWLJC0011_GET_CACHE_ERR.useraction=Address the error included in the message and retry.
 
-CWLJC0012_CLOSE_CACHE_ERR=CWLJC0012E: Error closing the {0} JCache: {1}.
+CWLJC0012_CLOSE_CACHE_ERR=CWLJC0012W: Error closing the {0} JCache: {1}.
```

Comment:
is it possible to say what process or entity encountered the unspecified error? eg, 

`<x> encountered the following error when it tried to close the {0} JCache:`

Reply:
See suggestions below.